### ISSUE TITLE
[Issue] Status showing garbled 

### ISSUE THREAD

**Issue by xMikux** [nanite/Simple-Discord-Rich-Presence#28]
### Minecraft Version

1.18.2

### Modloader

Forge 40.1.92

### Mod version

3.0.3-build.26+mc1.18.2

### Repeatable?

- [X] Yes

### Link to log files

https://mclo.gs/CFb2qoz

### Steps to reproduce the issue

1. Open StoneBlock3 1.5.0 with game language traditional chinese.
2. See your Discord Profile

### Description of the issue you are having

![image](https://user-images.githubusercontent.com/26039249/209296745-24388d63-9c5e-46e9-8be6-af15b8ecac7c.png)

When using Simple Discord Rich Presence on traditional chinese, it will show up garbled info.

### Information

- [X] I have provided as much information as possible

**Comment by Sunekaer** [comment]
Yeah idk how to fix this, because when i test on my pc it shows correctly
